Dockerfile

```diff
@@ -1,7 +1,13 @@
-FROM alpine:3.5
+FROM golang:alpine AS binarybuilder
```

Comment:
golang:alpine is docker image of alpine:latest platform of official golang latest stable build version.   
 eg: current golang:alpine == (alpine-3.8, go-1.10.3)